File: template.swift
#!/usr/bin/env swift

import Foundation
import RegexBuilder

let start = Date()
let input = try String(contentsOfFile: #file.replacingOccurrences(of: ".swift", with: ".input"))
var result1 = 0
var result2 = 0

// TODO: Good luck

print("part1", result1)
print("part2", result2)
print("elapsed time in seconds:", -start.timeIntervalSinceNow)

File: template.r
#!/usr/bin/env Rscript

start = Sys.time()
library("this.path")
input = read.delim(gsub('r$', 'input', this.path()))
result1 = 0
result2 = 0

# TODO: Good luck

paste("part1", result1, result1 == 0)
paste("part2", result2, result2 == 0)
paste("elapsed time in seconds:", Sys.time() - start)
